go-controller/hybrid-overlay/pkg/controller/node.go

```diff
@@ -90,6 +90,7 @@ func NewNode(
 	nodeInformer cache.SharedIndexInformer,
 	localPodInformer cache.SharedIndexInformer,
 	eventHandlerCreateFunction informer.EventHandlerCreateFunction,
+	isHONode bool,
```

Comment:
I guess you would have to pass this one to `newNodeController`

Reply:
fixed